src/app/ApplicationTemplate.Shared.Views/Startup.cs

```diff
@@ -149,82 +140,86 @@ void OnStateChanged(bool canNavigateBackOrCloseModal)
 					: AppViewBackButtonVisibility.Collapsed;
 			}
 		}
+#endif
+//+:cnd:noEmit
 	}
 
 	/// <summary>
 	/// Adds SystemNavigation's back requests to the IBackButtonManager. Those requests are usually sent via the top bars back buttons.
 	/// </summary>
 	private async Task AddSystemBackButtonSource(IServiceProvider services)
 	{
-		var dispatcher = services.GetRequiredService<CoreDispatcher>();
+//-:cnd:noEmit
+#if __ANDROID__ || __IOS__
+		var dispatcherQueue = services.GetRequiredService<DispatcherQueue>();
 		var backButtonManager = services.GetRequiredService<IBackButtonManager>();
-		await dispatcher.RunAsync((CoreDispatcherPriority)CoreDispatcherPriority.High, AddSystemBackButtonSourceInner);
+		await dispatcherQueue.RunAsync(DispatcherQueuePriority.High, AddSystemBackButtonSourceInner);
 
 		// Runs on main thread.
 		void AddSystemBackButtonSourceInner()
 		{
 			var source = new SystemNavigationBackButtonSource();
 			backButtonManager.AddSource(source);
 		}
-	}
 #endif
+//+:cnd:noEmit
+	}
 
 	protected override ILogger GetOrCreateLogger(IServiceProvider serviceProvider)
 	{
 		return serviceProvider.GetRequiredService<ILogger<Startup>>();
 	}
 
-#if false
 	private void SetStatusBarColor(IServiceProvider services)
 	{
 //-:cnd:noEmit
 #if __ANDROID__ || __IOS__
-//+:cnd:noEmit
 		var dispatcher = services.GetRequiredService<IDispatcherScheduler>();
 
-		// These are pages with a different background color, needing a different status bar color
-		Type[] vmsAlternateColor =
+		// These are pages with a different background color, needing a different status bar color.
+		var vmsAlternateColor = new Type[]
 		{
 			typeof(OnboardingPageViewModel),
 			typeof(LoginPageViewModel),
 			typeof(ForgotPasswordFormViewModel),
 			typeof(ResetPasswordPageViewModel),
-			typeof(SentEmailConfirmationPageViewModel)
+			typeof(SentEmailConfirmationPageViewModel),
 		};
 
 		services
 			.GetRequiredService<ISectionsNavigator>()
 			.ObserveProcessedState()
 			.ObserveOn(dispatcher)
-			.Subscribe(onNext: state =>
-			{
-				var currentVmType = state.CurrentState.GetViewModelType();
+			.Subscribe(
+				onNext: state =>
```

Comment:
I don't we why we would want to add an indentation level.

Reply:
Looked cleaner to me but I'm not gonna fight over it! I'll change it back ahah!